stdlib/LinearAlgebra/src/symmetric.jl

```diff
@@ -574,6 +574,12 @@ function _factorize(A::HermOrSym{T}; check::Bool=true) where T
     end
 end
 
+logabsdet(A::RealHermSymComplexHerm) = ((l, s) = logabsdet(_factorize(A; check=false)); return real(l), s)
+logabsdet(A::Symmetric{<:Real}) = logabsdet(_factorize(A; check=false))
```

Comment:
The same `real`ification should then probably apply here and in lines 581 and 584, shouldn't it? Or are they guaranteed to yield real return types?

Reply:
Seems like this is guaranteed to be real already.